docs/tests.rst

```diff
@@ -110,6 +110,57 @@ Via Docker with a graphical instance of Firefox (macOS)
 
   $ docker-compose -f docker-compose.test.yml run --rm acceptance
 
+How to Debug the Selenium Tests on Docker
+-----------------------------------------
```

Comment:
This logically belongs _after_ the section on how to write tests. Also, since we don't support running the `acceptance` natively, there's no point in specifying "on Docker".